soomer.py

```diff
@@ -1,31 +1,33 @@
 import discord
 import logging
-# logging.basicConfig(Level=logging.WARNING)
+import re
+import sqlite3
+
+conn = sqlite3.connect('emojilytics.db')
```

Comment:
You should handle the db schema outside of the code itself.

Consider using a migration framework like this one: https://github.com/clutchski/caribou